modules/nw-ovn-kubernetes-logs.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-troubleshooting-sources.adoc
+
+:_content-type: PROCEDURE
+[id="nw-ovn-kubernetes-logs_{context}"]
+= OVN-Kubernetes logs
```

Comment:
This title should use the procedure module format from the mod docs guidelines.

Reply:
Rewritten to reflect this is a procedure